### DISCUSSION TITLE
How do GitHub Discussions work?

### DISCUSSION THREAD

**Discussion by deerskindoll** [deerskindoll/git_gud#1] [Announcements]
## 👋 Welcome!
  Lorem ipsum all the way!

**Comment by deerskindoll** [comment]
- [x] create a discussion
- [ ] test discussion features
- [ ] ???
- [ ] success

<details>
<summary>Details</summary>

collapsible content. neat!

</details>

**Reply by deerskindoll** [reply]
This is a response to the first post.

**Reply by nickboldt** [reply]
with a followup!

**Comment by deerskindoll** [comment]
This is an entirely new comment with a task list and collapsible content.

- [ ] draft
- [ ] dev review
- [ ] peer review
- [ ] done

<details>
<summary>Details</summary>

details details details

</details>

**Reply by nickboldt** [reply]
is this better than a JIRA with subtasks?

**Reply by deerskindoll** [reply]
That's what we're trying to find out.

![cute-puppy](https://github.com/deerskindoll/git_gud/assets/80100533/5fc8bc93-71e0-4c51-a3ca-2398c791fd93)
